nemo/collections/nlp/models/language_modeling/bert_lm_model.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             checkpoint_file=cfg.language_model.lm_checkpoint,
         )
 
-        self.hidden_size = self.bert_model.hidden_size
+        self.hidden_size = self.bert_model.config.hidden_size
```

Comment:
This hidden_size looks like to just be used in the next line. So how about just use the self.bert_model.config.hidden_size directly or at least not to store it in the self as it is not used?

Reply:
done